kotlinx-coroutines-debug/test/SanitizedProbesTest.kt

```diff
@@ -63,6 +63,7 @@ class SanitizedProbesTest : DebugTestBase() {
                     "\t(Coroutine creation stacktrace)\n" +
                     "\tat kotlin.coroutines.intrinsics.IntrinsicsKt__IntrinsicsJvmKt.createCoroutineUnintercepted(IntrinsicsJvm.kt:116)\n" +
                     "\tat kotlinx.coroutines.intrinsics.CancellableKt.startCoroutineCancellable(Cancellable.kt:23)\n" +
+                    "\tat kotlinx.coroutines.BuildersKt__Builders_commonKt.async\$default(Builders.common.kt)\n" +
```

Comment:
It seems like the actual frame (one with line numbers) points to the wrong line.
Typically, it's worth reporting it to our bugtracker (subsystem: `Backend. JVM. Coroutines`, assignee -- Ilmir)